spec/spec_helper.rb

```diff
@@ -23,8 +24,47 @@
 module RSpecMixin
   include Rack::Test::Methods
 end
+begin
```

Comment:
Feels a bit out of context here...
Would it be more suitable to move some of this stuff into Rakefile instead?